samples/opened-editors-list/package.json

```diff
@@ -1,16 +1,15 @@
 {
-      "name": "opened-editor-list",
-      "publisher": "theia",
-      "keywords": [
+    "name": "opened-editors-list",
+    "publisher": "theia",
```

Comment:
eclipse-che ?

Reply:
Do you mean `@eclipse-che/opened-editors-list` ??

If yes, we need to separate samples from real plugins stored in https://github.com/eclipse/che-theia/tree/master/plugins

`@eclipse-che/samples/opened-editors-list` ?

Reply:
> @eclipse-che/samples/opened-editors-list ?

I think no - https://docs.npmjs.com/about-scopes
`@eclipse-che` - it's scope of the package, `opened-editors-list` - node package name.

Reply:
I was referencing the "publisher" : eclipse-che instead of theia

Reply:
Done